src/angular/styles/_typography.scss

```diff
@@ -228,11 +228,14 @@
   --sbb-fieldset-legend-nested-font-size: #{sbb.pxToRem(20)};
 
   // Forms
-  --sbb-form-background-color-readonly: var(--sbb-color-milk);
-  --sbb-form-border-color-readonly: var(--sbb-color-aluminum);
-  --sbb-form-field-width: #{sbb.pxToRem(320)};
-  --sbb-form-input-color: var(--sbb-color-charcoal);
+  --sbb-form-background-color-default: var(--sbb-color-background);
+  --sbb-form-background-color-disabled: var(--sbb-color-milk);
+  --sbb-form-border-color-default: var(--sbb-color-graphite);
```

Comment:
We should not change the color in the standard mode. The original color was charcoal for standard mode. Please change it back.

For lean mode it should be granite, if I see it right. So this has to be overridden in the `where(.sbb-lean)` block.